eagle-stores/src/pages/Cart/components/CartCard.css

```diff
@@ -0,0 +1,126 @@
+.cart__container {
+    width: 100vw;
+    padding: 2rem 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+}
+
+.content__heading {
+    padding: 1rem 0;
+}
+
+.cart {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.cart__horizontal {
+    width: 30rem;
+    height: 22rem;
+    display: flex;
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
```

Comment:
You can use CSS variables here as well

Reply:
This is rgba and i am using it once so that's why I didn't used variables

Reply:
Okay no issues ✅